src/xpcc/ui/display/buffered_graphic_display.hpp

```diff
@@ -55,6 +55,10 @@ namespace xpcc
 		static_assert((Height % 8) == 0, "height must be a multiple of 8");
 
 	public:
+		static constexpr uint16_t display_buffer_width = Width;
+		static constexpr uint16_t display_buffer_height = Height / 8;
```

Comment:
Could you Capitalize this?
I find this super confusing especially in combination with `this->display_buffer_width`, which makes it look like a runtime variable.